1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,11 +14,11 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
-names[4] = names[4].substring();
+names[0] = names[0].substring(0, 5);
```

Comment:
Hi Meysam just wanted to let you know in this line of code the indexes you used will print the name Tamzi instead of Tamzin the indexes for the print of full first name are (0, 6)